peachjam/views/legislation.py

```diff
@@ -277,12 +277,14 @@ def get_timeline(self):
         return timeline
 
     def get_child_documents(self):
-        docs = (
+        docs_ids = (
             self.model.objects.filter(parent_work=self.object.work)
             .distinct("work_frbr_uri")
-            .order_by("work_frbr_uri", "-date")
+            .order_by("work_frbr_uri")
         )
         # now sort by title
+        docs = self.model.objects.filter(pk__in=docs_ids).order_by(
+            "-date", "-frbr_uri_number"
+        )
```

Comment:
added this for the sorting. It was not working well because of the distinct work_frbr_uri. Its still not very good. as you can see from the screenshot. I think we need a better solution just for sorting.

Reply:
sorting by work__frbr_uri_date is much better
![image](https://github.com/user-attachments/assets/4324d0a8-f459-4e1a-88da-7c32883b7de1)